pageserver/src/layered_repository/delta_layer.rs

```diff
@@ -607,12 +622,9 @@ impl DeltaLayerWriter {
         //
         // Note: This overwrites any existing file. There shouldn't be any.
         // FIXME: throw an error instead?
-        let path = conf.timeline_path(&timelineid, &tenantid).join(format!(
-            "{}-XXX__{:016X}-{:016X}.temp",
-            key_start,
-            u64::from(lsn_range.start),
-            u64::from(lsn_range.end)
-        ));
+        let path =
+            DeltaLayer::temp_path_for(conf, timelineid, tenantid, key_start, lsn_range.clone());
```

Comment:
We can just pass `lsn_range` by reference and remove the `clone` here.

Reply:
thank you, fixed